src/labels/CustomLabels.labels

```diff
@@ -56,6 +56,14 @@
         <shortDescription>CannotDelete</shortDescription>
         <value>Record cannot be deleted because children records exist. Delete children first.</value>
     </labels>
+    <labels>
```

Comment:
Was this verified by Julie? I'm curious why we're adding a more specific label here and not using the "CannotDelete" label. I think being specific is better, but it's not consistent with what we've done in the past so just want to make sure everyone is in agreement this is what we want to do here.

@lifewithryan thoughts?

Reply:
@skosgi1 You removed the label everywhere but forgot to do it here :)

Reply:
@robertbwatson  😂